gatsby-node.ts

```diff
@@ -94,3 +109,31 @@ export const onCreateWebpackConfig: GatsbyNode['onCreateWebpackConfig'] =
       });
     }
   };
+
+export const createResolvers = ({ createResolvers }) => {
+  createResolvers({
+    ContentfulBlogPost: {
```

Comment:
Idea: Would it make sense to also add the `path` here? To not have 2 places where we modify the type.